### PR TITLE
Search for personnel code if unrecognised msisdn

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@NicolaasDieBaas @imsickofmaps Please review.

File: src/utils.js [hunk 1]
```diff
@@ -679,9 +679,15 @@ go.utils = {
...
-                return content === '12345';
+        var params = {
+            "details__personnel_code": content
+        };
+        return go.utils
+            .service_api_call('identities', 'get', params, null, 'search/', im)
+            .then(function(json_get_response) {
+                var contacts_found = json_get_response.data.results;
+                // Return the first contact's id
+                return contacts_found.length > 0;
```
Comment (imsickofmaps) [inline src/utils.js:690]: this isn't returning what the comment says

File: test/fixtures.js [hunk 2]
```diff
@@ -70,12 +79,47 @@ module.exports = function() {
...
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [{
+                        "url": "http://localhost:8000/api/v1/identities/cb245673-aa41-4302-ac47-00000000001/",
+                        "id": "cb245673-aa41-4302-ac47-00000000001",
+                        "version": 1,
+                        "details": {
+                            "default_addr_type": "msisdn",
+                            "addresses": "msisdn:+2347030010001"
```
Comment (imsickofmaps) [inline test/fixtures.js]: This the old style contact object

**Comment by imsickofmaps** [comment]
Minor changes as discussed then :+1:

**Pull request merged**